tests/website.py

```diff
@@ -16,6 +16,9 @@ def _pre_setup_10_website(self):
         self.deployment.add("haproxy")
         self.deployment.relate("jenkins:website", "haproxy:reverseproxy")
         self.deployment.expose("haproxy")
+        self.deployment.add("nrpe")
+        self.deployment.relate("jenkins:nrpe-external-master",
+                               "nrpe:nrpe-external-master")
```

Comment:
Sure I can make a new test for it I put it here in an attempt to avoid adding more time to the already pretty long test cycle with amulet.